### ISSUE TITLE
About delta time when timestamp of datapoints contains a value of a unit millseconds or less when calculating Rate at 3.0 branch

### ISSUE THREAD

**Issue by yyang800** [OpenTSDB/opentsdb#2209]
Is this implementation intended to normalize the target data point?
https://github.com/OpenTSDB/opentsdb/blob/628bc6ee962293bb94ff531be671d9004e49db50/core/src/main/java/net/opentsdb/query/processor/rate/RateNumericIterator.java#L190-L193
If the timestamp of a data point contains a value in units of millseconds or less, it looks like the delta time may be calculated very large compared to the actual elapsed time.

https://github.com/OpenTSDB/opentsdb/blob/628bc6ee962293bb94ff531be671d9004e49db50/core/src/main/java/net/opentsdb/query/processor/rate/RateNumericIterator.java#L184-L196

Example
```
prev: 1640077200200L
next: 1640077260100L
expected: 59.9
actual: 1.000000588E8
```

**Comment by manolama** [comment]
Yeah @yyang800, that is a silly bug on my part on L191, thank you. Mind giving me a PR to change the `*=` to `+=` please? And thank you for catching it.
